app/code/Magento/InventoryAdminUi/Test/Mftf/Test/AdminBackordersEnablesOnConfigurationPageAndAppliedToGroupedProductCustomStock.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<tests xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/testSchema.xsd">
+    <test name="AdminBackordersEnablesOnConfigurationPageAndAppliedToGroupedProductCustomStock" extends="AdminCreateGroupedProductCustomStock">
```

Comment:
This does not seem like meaningful inheritance. Would be great to refactor this down the road.

Reply:
Will be fixed in next PR